components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -146,10 +125,15 @@ export default function () {
                         </a>
                     </span>
                 }
-                checked={isIncrementalPrebuildsEnabled}
-                disabled={isLoading}
```

Comment:
I wonder if we should keep the disabled state of checkboxes, e.g. for when `project` isn't loaded yet? 🤔 This could avoid having interactive checkboxes in the wrong state (e.g. everything seems disabled while we're loading the actual project settings.)

Not super important though. 👍

Reply:
I'd rather work on making loading a single project faster if that's really an issue